src/constants.ts

```diff
@@ -23,19 +23,19 @@ export const getControlTreatmentsWithConfig = (splitNames: string[]): SplitIO.Tr
 };
 
 // Action types
-export const SPLIT_READY = 'SPLIT_READY';
+export const SPLIT_READY: string = 'SPLIT_READY';
```

Comment:
Remove all the unnecessary typings. Try to not be redundant with type inference.